ast/src/main/java/com/graphicsfuzz/common/glslversion/ShadingLanguageVersion.java

```diff
@@ -369,4 +369,9 @@ static ShadingLanguageVersion webGlFromVersionString(String versionString) {
    */
   boolean supportedShaderMemoryControlFunctions();
 
+  /**
+   * GLSL versions 4.6 and ESSL versions 3.2+ support push_constants for vulkan.
```

Comment:
I would add a + after 4.6 (even if 4.7 doesn't exist yet), and change "vulkan" to "Vulkan".

Reply:
Done